cmd/backup/main.go

```diff
@@ -11,6 +11,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// This test case is used to check whether BACKUP/RESTORE
+// compatible with different features (eg. async-commit/one-pc)
+// by checking whether we can restore the whole database no matter when we backup
+// Maintainer: longfangsong <longfangsong@icloud.com>
+//
+// example command to run this case:
+// with a tidb server running under 127.0.0.1:4000, with async-commit and one-pc on:
+// ./bin/backup -tidb-server 127.0.0.1:4000 -async-commit 1 -one-pc 1
+//
+// This case is supposed to run forever, until an error occur or got killed
+// This case should tolerant with all kinds of nemesis
```

Comment:
The container will be recreated when the instance is killed and the data stored in /tmp/xxx will be removed.
It seems that the nemesis like `random_kill` should not be used.

Reply:
If things like `random_kill` is on, we should keep the backup file in externel storage like s3, I'll add a note for it.